sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -138,6 +138,7 @@ public partial class ContainerRegistryClient
         public ContainerRegistryClient(System.Uri endpoint, Azure.Core.TokenCredential credential) { }
         public ContainerRegistryClient(System.Uri endpoint, Azure.Core.TokenCredential credential, Azure.Containers.ContainerRegistry.ContainerRegistryClientOptions options) { }
         public virtual System.Uri Endpoint { get { throw null; } }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
```

Comment:
Could you provide some context on why this needs to be public? If this behavior for a specific client call is intended to be exposed to customers, I'd expect it to be exposed via some client options or parameter on the specific methods that require it. If for the client itself, I'd expect it to be internal.

Reply:
Sure, there are a couple of reasons.  

The first is that our team is moving toward a strategy of giving lower-level access to library users by exposing the Pipeline as a public member on the client.  This is generated automatically in DPG clients, so this is following that precedent, just implementing it manually in this case.

The second is that this scenario is sufficiently uncommon, and not fully documented by the service team, that we don't want to expose it in a supported API on a GA client, but we do want to make it available to callers who understand the details and are willing to use an advanced API to accomplish it.

Reply:
Discussed offline.